carbon-design-system/carbon-addons-iot-react
# style(card-editor): fix rtl issues on threshold form items

Closes #3068 

**Summary**

- fixes the minor RTL issues with the threshold form items

**Change List (commits, features, bugs, etc)**

- fix RTL issues

**Acceptance Test (how to verify the PR)**

- https://deploy-preview-3082--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-cardeditor-cardeditformitems-thresholdformitem--example-with-initial-values
- turn on RTL knob
- confirm design matches LTR

**Regression Test (how to make sure this PR doesn't break old functionality)**

- n/a



**Things to look for during review**
